git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
Well done, sir!

BTW I just realized that `odt2txt` tries to determine the encoding based on the current Win32 Console. Maybe we want to pass the `--encoding=UTF-8` option here? It seems to work better for me.

Also: we will have to measure the delta of the installer first before updating the commit message accordingly.